components/07-call-to-action/call-to-action.config.json

```diff
@@ -0,0 +1,18 @@
+{
+	"title": "Form",
+    "name": "form",
```

Comment:
Hi Pamela
I believe this (title and name values) may be causing the display issue, because it has the same values as components/02-components/05-form/form.config.json

Reply:
One more thought....while I'm at it.
Please update all values within all files to reflect this component name. 
So swap out the values/strings in the json file, readme, scss comments, etc.

Reply:
Right. I updated the name everywhere...but I didn't save all the files before pushing the first time. Oops!

Just did that and also moved call-to-action component underneath main component section... I think that fixed everything. At least it looks like it on my end. 

I have some Draper work I need to do for KC to get ready for our stakeholder meeting this afternoon so it'll probably be tomorrow until I get back to this.  Thanks for your patience!

Reply:
This has been fixed. I hadn't SAVED the updates to the config,json file, so it didn't get pushed originally.